### DISCUSSION TITLE
Health Check start_interval "not" supported by Synology

### DISCUSSION THREAD

**Discussion by cr0n76** [immich-app/immich#10519] [Q&A] [answered]
Hi,

unfortunately Docker is severely outdated on Synology NASs. An appropriate update is expected with the next OS update (that is "just around the corner") but status quo is that docker compose is version 2.9ish and "start_interval" is introduced in 2.20ish. 
No Synology user can do anything against this then to wait for the next OS update.

So first of all I want to make you aware that this might increase support requests, as currently all Synology users who overtake the healthcheck might run into issues without being aware of the reason of the issue.

And I wanted to ask if you have suggestions how to deal with it?
Should I just remove it from the .yml and accept that the first five minutes (or 30s?) the database might be shown unhealthy? Do anyone of you have other ideas for workarounds?

**Comment by bo0tzz** [comment]
You can just remove it.

**Reply by cr0n76** [reply]
Yeah, have done this and its working fine. I am jut curious if the health status is used somewhere by e.g. immich-server?

**Reply by mmomjian** [reply, marked as answer]
It won’t remove the healthcheck. It just means it will take longer to initially report healthy. It’s not an issue
